src/layouts/Default.vue

```diff
@@ -134,10 +134,10 @@ export default {
     return {
       isMenuActive: false,
       services: [
-        { name: "Lab", route: "/docs/services/lab" },
+        { name: "Computer Lab", route: "/docs/services/lab" },
         { name: "Printing", route: "/docs/services/lab/printing" },
-        { name: "Web Hosting", route: "/docs/services/web" },
-        { name: "Application Hosting", route: "/docs/services/webapps" },
+        { name: "Personal Website Hosting", route: "/docs/services/web" },
```

Comment:
Should the name attributes here match the title on the MD file? That is, "Web hosting (personal)"

Reply:
These are the items that are in the nav bar at the top, we wanted to increase the clarity of what each option linked to the in docs.

Reply:
I wanted to clearly differentiate group hosting from personal hosting so people know which one to pick at a glance.

Reply:
Yeah I think either is fine because it's still clear, the only reason to make them match the page title would be consistency

Reply:
Resolving this, seems like a good rationale